### ISSUE TITLE
Can't disable value-argument/parameter-comment

### ISSUE THREAD

**Issue by ShindongLee** [pinterest/ktlint#2667]
I've read issue https://github.com/pinterest/ktlint/issues/2445 which is resolved but I am having same experiences.

## Expected Behavior

Rule value-argument-comment and value-parameter-comment is disabled

## Observed Behavior

ktlintFormat / ktlintCheck fails with error:

```
A comment in a 'value_argument_list' is only allowed when placed on a separate line (cannot be auto-corrected)
A comment in a 'value_parameter_list' is only allowed when placed on a separate line (cannot be auto-corrected)
```

## Steps to Reproduce

.editorconfig
```
[*.{kt,kts}]
charset = utf-8
end_of_line = lf
indent_size = 4
indent_style = space
trim_trailing_whitespace = true
insert_final_newline = true
max_line_length = 180
tab_width = 4
ktlint_standard_value-argument-comment = disabled
ktlint_standard_value-parameter-comment = disabled
```

code
```kotlin
something.function(
      a = a,
      b = b,
      c = c, // comment <-- A comment in a 'value_argument_list' is only allowed when placed on a separate line (cannot be auto-corrected)
)

data class Something(
    val a: A,
    val b: B,
    val c: C, // comment <-- A comment in a 'value_parameter_list' is only allowed when placed on a separate line (cannot be auto-corrected)
)
```

## Your Environment

* Version of ktlint used: 1.2.1
* Relevant parts of the `.editorconfig` settings
```
ktlint_standard_value-argument-comment = disabled
ktlint_standard_value-parameter-comment = disabled
```
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): ktlint-gradle-plugin 12.1.1
* Version of Gradle used (if applicable): 8.2
* Operating System and version: Mac OS Sonoma 14.3.1

**Comment by ShindongLee** [comment]
never mind,

I am using gradle multi module, and I moved the ktlint block in build.gradle.kts

```kotlin
ktlint {
    version.set("1.2.1")
}
 ```
 
 into subproject
 ```kotlin
subprojects {
    ktlint {
        version.set("1.2.1")
    }
}
 ```
 
 and it disabled the rules.
 
 Thank you.

**Comment by paul-dingemans** [comment]
Nice you solved it yourselves. Enjoy using ktlint!
